### ISSUE TITLE
Warnings during jack build

### ISSUE THREAD

**Issue by andreya108** [tony19/logback-android#128]
logback has dependency on com.google.android:android:2.1_r1

When switching to build tools 24 and using jack compiler in build log warnings like the following appear:

WARNING: Ignoring Android API artifact com.google.android:android:2.1_r1 for appDebug 
WARNING: Ignoring Android API artifact com.google.android:android:2.1_r1 for appRelease

**Comment by pillisan42** [comment]
you have to add exclusion on group 
```
compile ('com.github.tony19:logback-android-core:<version>') {
        exclude group: 'com.google.android'
    }
    compile ('com.github.tony19:logback-android-classic:<version>') {
        exclude group: 'com.google.android'
    }
```

I hope this will help some people 

source:  [new-warning-when-building-android-app-with-gradle](http://stackoverflow.com/questions/37324917/new-warning-when-building-android-app-with-gradle)

**Comment by andreya108** [comment]
That is what I missed! Thanks!

**Comment by Nursik91** [comment]
Hi! I faced this warning too. When I add exclusion on group how was mentioned above I see next message "Could not find method com.github.tony19:logback-android-classic:1.1.1-3" and it doesn't build.
This what I did:
compile 'com.github.tony19:logback-android-classic:1.1.1-3'{
        exclude group: 'com.google.android'
    }
compile 'com.github.tony19:logback-android-core:1.1.1-3'{
        exclude group: 'com.google.android'
    }

Maybe I did something wrong?
